src/components/VmDetails/GridComponents.js

```diff
@@ -19,7 +19,7 @@ Grid.propTypes = {
 
 const Row = ({ className = '', children, style, ...props }) => {
   const cn = `${styles['grid-row']} ${className}`
-  return <div className={cn} style={style}>{children}</div>
+  return <div className={cn} style={style} {...props}>{children}</div>
```

Comment:
I hit an error at runtime when I had a non-HTML attribute name in `...props`. May be worth specifying the `id` directly to protect against random prop name errors in the future.